scenarios/Assistants/api-in-a-box/personal_finance/assistant-personal_finance.ipynb

```diff
@@ -4,9 +4,57 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Personal Financial Assistant\n",
+    "# Personal Financial Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Time\n",
+    "\n",
+    "You should expect to spend 10 minutes running this sample. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## About this example\n",
+    "\n",
+    "{{TODO: Include a paragraph or two explaining what this example demonstrates, who should be interested in it, and what you need to know before you get started.}}\n",
+    "\n"
```

Comment:
Needs a description

Reply:
# Personal Financial Assistant

## Objective
This notebook demonstrates the following:

- Function Calling with Yfinance to get latest stock prices. Summarization of user provided article. Extract country info from article, extract country, capital and other aspects, and call an API to get more information about each country.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant)
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 10-15 minutes running this sample.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/ca2359e0ec502e2143942885df0ad0c2ebcbfa47

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/9a6b45d1e81a272bfed87e728d2b40ece73ce97e